app/controllers/api/v1/invoice_items_search_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Api::V1::InvoiceItemsSearchController < ApiBaseController
+  respond_to :json, :xml
+
+  def index
+    if invoice_item_params[:id]
+      respond_with InvoiceItem.where("id = ?", invoice_item_params[:id])
+    elsif invoice_item_params[:quantity]
+      respond_with InvoiceItem.where("quantity = ?", invoice_item_params[:quantity])
+    elsif invoice_item_params[:unit_price]
+      respond_with InvoiceItem.where("unit_price = ?", invoice_item_params[:unit_price])
+    elsif invoice_item_params[:item_id]
+      respond_with InvoiceItem.where("item_id = ?", invoice_item_params[:item_id])
+    elsif invoice_item_params[:invoice_id]
+      respond_with InvoiceItem.where("invoice_id = ?", invoice_item_params[:invoice_id])
+    elsif invoice_item_params[:created_at]
+      respond_with InvoiceItem.where("created_at = ?", invoice_item_params[:created_at])
+    elsif invoice_item_params[:updated_at]
+      respond_with InvoiceItem.where("updated_at = ?", invoice_item_params[:updated_at])
+    else
+      respond_with InvoiceItem.where(invoice_item_params)
```

Comment:
This comment applies to all the search controllers. 
--What is the catch-all doing if we have a line for each column? 
--Based on Postman testing, I don't think we necessarily need a separate line for every column. That said, it is probably a good idea to do it this way until we have everything properly tested. Either way, we should refactor for uniformity across our record endpoints.